### ISSUE TITLE
Implement Timer for someone leaving the Game

### ISSUE THREAD

**Issue by web3technologies** [realm-of-ra/mancala#55]
If someone doesn't make a move for n number of seconds we need to forfeit the game.

**Comment by TAdev0** [comment]
hi @web3technologies  @okhaimie-dev  may i work on this issue? :)

**Comment by web3technologies** [comment]
@TAdev0 assigned to you!

**Comment by TAdev0** [comment]
@web3technologies  its the first I use dojo
I cannot compile the repo, because of this errror:
```
error: compiler plugin could not be loaded `dojo_plugin v0.3.11 (git+https://github.com/dojoengine/dojo?tag=v0.3.11#1e651b5d4d3b79b14a7d8aa29a92062fcb9e6659)`
```

any idea?

**Comment by web3technologies** [comment]
2 things. 

1. Are you using scarb build or sozo build? (Dojo requires the use of sozo not scarb) run sozo build and sozo test
2. What version of scarb are you using? The scarb.toml is using cairo-version = "2.5.4"

**Comment by TAdev0** [comment]
ok thanks i need to use `sozo build` (first time as well)

yeah i saw its 2.5.4, i'm using ASDF, i use the latest by default but created a .tool-versions locally to work with 2.5.4 on the repo

**Comment by web3technologies** [comment]
In dojo 0.6.0 there are many issues with the language server. Cartridge has said when they release 0.7.0 soon there will be many updates to the language server to fix this. When this is released we will be migrating over to 0.7.0 but for now we have to manage the errors.

**Comment by TAdev0** [comment]
got it, thanks!
